metadata.rb

```diff
@@ -4,17 +4,16 @@
 license          'Apache-2.0'
 description      'Installs/Configures Splunk Servers and Forwarders'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.21.1'
+version          '2.21.2'
 
 source_url       'https://github.com/cerner/cerner_splunk'
 issues_url       'https://github.com/cerner/cerner_splunk/issues'
 
-chef_version     '~> 12.4' if respond_to?(:chef_version)
+chef_version     '>= 12.7', '< 15' if respond_to?(:chef_version)
```

Comment:
minor, chef_version was [introduced in 12.6](https://github.com/chef/chef/commit/98a1793e95e7f89b5206883528a0d00c209838f0#diff-e7f90eec970c3a67a738e9000abf630c) so since we're requiring 12.7 or greater we don't need the `if respond_to?()` check anymore.

Reply:
removed in: accc160